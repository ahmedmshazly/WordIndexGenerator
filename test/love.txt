"No. That's a fantasy, honey. Love and marriage are about work and
compromise. They're about seeing someone for what he is, being
disappointed, and deciding to stick around anyway. They're about
commitment and comfort, not some kind of sudden, hysterical
recognition."

"That's not what I want. Disappointment and comfort is not what I want."

"Why not? Because you expect it to be magical and mystical? Because
you don't want to work?"

"Why can't it be magical? Why can't it be mystical?"

"Because if you count on magic and mysticism, Emilia, then as soon as
shit happens, as soon as life interferes, as soon as your stepson
treats you badly, or your husband's ex-wife has a fit about something,
or your baby dies, as soon as life happens, the magic will disappear
and you'll be left with nothing. You can't count on magic,
Emilia. Trust me, I know. Sweetheart, little girl, you can't count on
magic."

- Ayelet Waldman, Love and Other Impossible Pursuits
