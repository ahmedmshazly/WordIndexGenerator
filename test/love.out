can't           [4]: 12, 18, 19
magic           [4]: 14, 17, 18, 20
soon            [4]: 14, 15, 17
count           [3]: 14, 18, 19
why             [3]: 9, 12
comfort         [2]: 4, 7
emilia          [2]: 14, 19
happens         [2]: 15, 17
life            [2]: 15, 17
love            [2]: 1, 22
magical         [2]: 9, 12
mystical        [2]: 9, 12
that's          [2]: 1, 7
they're         [2]: 2, 3
anyway          [1]: 3
around          [1]: 3
ayelet          [1]: 22
baby            [1]: 17
badly           [1]: 16
being           [1]: 2
commitment      [1]: 4
compromise      [1]: 2
deciding        [1]: 3
dies            [1]: 17
disappear       [1]: 17
disappointed    [1]: 3
disappointment  [1]: 7
don't           [1]: 10
ex-wife         [1]: 16
expect          [1]: 9
fantasy         [1]: 1
fit             [1]: 16
girl            [1]: 19
has             [1]: 16
honey           [1]: 1
husband's       [1]: 16
hysterical      [1]: 4
impossible      [1]: 22
interferes      [1]: 15
kind            [1]: 4
left            [1]: 18
little          [1]: 19
marriage        [1]: 1
mysticism       [1]: 14
nothing         [1]: 18
pursuits        [1]: 22
recognition     [1]: 5
seeing          [1]: 2
shit            [1]: 15
someone         [1]: 2
something       [1]: 16
stepson         [1]: 15
stick           [1]: 3
sudden          [1]: 4
sweetheart      [1]: 19
treats          [1]: 16
trust           [1]: 19
waldman         [1]: 22
you'll          [1]: 18
